main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/companieshouse/chs.go/log"
+
+	"github.com/companieshouse/payments.api.ch.gov.uk/config"
+	"github.com/companieshouse/payments.api.ch.gov.uk/handlers/private"
+	"github.com/companieshouse/payments.api.ch.gov.uk/handlers/public"
+
+	"github.com/gorilla/pat"
+	"github.com/justinas/alice"
+)
+
+func main() {
+	log.Namespace = "payments.api.ch.gov.uk"
```

Comment:
probably worth putting this as a global var so all packages can access. or even put it in config

Reply:
Done

Reply:
Unit test's will be coming after this PR. Along with start script and make file.